curator-framework/src/main/java/org/apache/curator/framework/CuratorFramework.java

```diff
@@ -1,233 +1,238 @@
-/**
```

Comment:
My only guess is that it's some sort of whitespace issue?

On Fri, May 8, 2015 at 9:51 AM, Jordan Zimmerman <notifications@github.com>
wrote:

> In
> curator-framework/src/main/java/org/apache/curator/framework/CuratorFramework.java
> <https://github.com/apache/curator/pull/56#discussion_r29907085>:
>
> > @@ -1,233 +1,238 @@
> > -/**
>
> Any idea why this looks like a complete replacement?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/curator/pull/56/files#r29907085>.
>